miniapp/USERGUIDE.md

```diff
@@ -39,7 +39,6 @@ The SDK is configured via manifest meta-data, the configurable values are:
 | Base URL                     | String  | `com.rakuten.tech.mobile.miniapp.BaseUrl`              | ❌         | 🚫        |
 | App ID                       | String  | `com.rakuten.tech.mobile.ras.AppId`                    | ❌         | 🚫        |
 | RAS Project Subscription Key | String  | `com.rakuten.tech.mobile.ras.ProjectSubscriptionKey`   | ❌         | 🚫        |
-| Host App Version             | String  | `com.rakuten.tech.mobile.miniapp.HostAppVersion`       | ❌         | 🚫        |
 | Host App User Agent Info     | String  | `com.rakuten.tech.mobile.miniapp.HostAppUserAgentInfo` | ✅         | 🚫        |
 
 **Note:**  
```

Comment:
There is a note written for Host App Version, something like `All meta-data values must be string values, including the value for com.rakuten.tech.mobile.miniapp.HostAppVersion...`, is it okay to remove that line too?

Reply:
Also, removing the following line from USERGUIDE?

```
<meta-data
   android:name="com.rakuten.tech.mobile.miniapp.HostAppVersion"
   android:value="your_app_version" />
```